### DISCUSSION TITLE
Foundry, EIP 1559 Not Support

### DISCUSSION THREAD

**Discussion by avenbreaks** [0xPolygon/polygon-edge#1012] [General]
# [ Foundry Smartcontract ]
## Description
Trying to deploy smartcontract using Foundry but EIP-1559 Not Support 

## Your environment
- OS and version : Ubuntu
- The version of the Polygon Edge : v0.6.0 - v0.6.1

## Steps to reproduce
`forge update `

```
forge create --contracts <path/contracts> \
--rpc-url <localhostRPC> \
--private-key <priv-key> \
--constructor-args:Contract
```

## Expected behavior
Successful Deployed 

## Logs Foundry 
Foundry EIP-1559 Not Support

**Comment by laviniat1996** [comment]
Hi @avenbreaks ,

Unfortunately, Edge doesn't support EIP-1559.

**Reply by JekaMas** [reply]
I believe, we're currently working on it
1. introducing a common interface for different transactions https://github.com/0xPolygon/polygon-edge/pull/1004
2. EIP1559 itself

**Reply by laviniat1996** [reply]
@JekaMas Thank you for this info. I wasn't aware the a PR for this is in the works.
